src/lib/statistics/statistics_objects/min_max_filter.cpp

```diff
@@ -160,9 +160,9 @@ bool MinMaxFilter<T>::does_not_contain(const PredicateCondition predicate_condit
       return value >= max || value2 <= min;
     }
     case PredicateCondition::Like: {  // NOLINTNEXTLINE - clang-tidy doesn't like the else path of the if constexpr
-      // We do not handle the case of outlier min/max values. For example `USA%` is not pruned whenever just a single
-      // value with a lower case exists, because the default ASCII-based sorting places all upper case values before
-      // lower case values.
+      // We use the ascii collation for min/max filters. This means that lower case letters are considered larger than
+      // upper case letters. This can lead to a situation, where the USA% (e.g., JOB query XXXXX) is not pruned
```

Comment:
This text was not meant to be copied. :)

Since "larger" might be confusing when talking about lexicographical sorting (which is isn't here, but anyhow), we might better use "This means that lower case letters are sorted behind upper case letters.".
The XX was meant to be replaced.

[...] where LIKE 'USA%' [...]